File: jm-lambda-with-existing-s3-code/hello.py
# import json


def handler(event, context):
    # print("request: {}".format(json.dumps(event)))
    return {
        "statusCode": 200,
        "headers": {"Content-Type": "text/plain"},
        "body": f"Hello, CDK 1025! You have hit event={event}",
    }

File: java/s3-sns-sqs-lambda-chain/lambda/hello.js
exports.handler = async function(event) {
    console.log("request:", JSON.stringify(event, undefined, 2));
    return {
        statusCode: 200,
        headers: { "Content-Type": "text/plain" },
        body: `Hello, CDK! You've hit ${event.path}\n`
    };
};